src/handlers/shared/stop.ts

```diff
@@ -7,23 +7,17 @@ export async function stop(context: Context, issue: Context["payload"]["issue"],
 
   // is there an assignee?
   const assignees = issue.assignees ?? [];
-  if (assignees.length == 0) {
-    logger.error("No assignees found for issue", { issueNumber });
-    await addCommentToIssue(context, "````diff\n! You are not assigned to this task.\n````");
-    return { output: "No assignees found for this task" };
-  }
-
   // should unassign?
-  const shouldUnassign = assignees[0]?.login.toLowerCase() == sender.login.toLowerCase();
+  const whoToUnassign = assignees.find((assignee) => assignee?.login.toLowerCase() === sender.login.toLowerCase());
```

Comment:
```suggestion
  const usersToUnassign = assignees.find((assignee) => assignee?.login.toLowerCase() === sender.login.toLowerCase());
```

Reply:
Should we use a plural when it only returns a singular user? and given that `/stop` only unassigns the invoker and never others I think user singular is more appropriate

Reply:
You are correct I put plural by accident, singular makes sense.